.github/workflows/static_checks.yml

```diff
@@ -77,3 +77,15 @@ jobs:
       uses: actions/checkout@722adc6
     - name: Markdown lint
       run: bin/markdownlint-all
+  generate_chart_docs:
```

Comment:
Could you rename this to something like `chart_docs_diff` to follow `proto_diff` example above? So it's made clear that this doesn't actually perform anything new and it just checks there's no diff...